CHANGELOG.md

```diff
@@ -13,6 +13,13 @@
   This allows to have multiple json elements with the same name in
   different source files
 
+* The `lobster-codebeamer` tool now uses codebeamer api v3.
```

Comment:
Hi, we have made a release in the mean time, see https://github.com/bmw-software-engineering/lobster/releases.
Please move the changes up to section `0.9.18-dev`